packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -110,6 +125,10 @@ export class ParticleGenerator {
   private _instanceVertices: Float32Array;
   private _randomSeed = 0;
 
+  @ignoreClone
+  private _directionMin = new Vector3();
+  @ignoreClone
+  private _directionMax = new Vector3();
```

Comment:
Just temporary intermediate variables used in calculations？

Reply:
use private static variables instead